pkg/audit/manager.go

```diff
@@ -523,13 +529,40 @@ func (am *Manager) reviewObjects(ctx context.Context, kind string, folderCount i
 
 			results := ToResults(&augmentedObj.Object, resp)
 
-			if len(resp.Results()) > 0 {
-				err = am.addAuditResponsesToUpdateLists(updateLists, results, totalViolationsPerConstraint, totalViolationsPerEnforcementAction, timestamp)
+			// Expand object and review any resultant resources
+			allResps := []*rtypes.Responses{resp}
+			base := &mutationtypes.Mutable{
+				Object:    objFile,
+				Namespace: &ns,
+				Username:  "",
+				Source:    mutationtypes.SourceTypeGenerated,
+			}
+			resultants, err := am.expansionSystem.Expand(base)
+			if err != nil {
+				am.log.Error(err, "Unable to expand object", "objName", objFile.GetName())
+				continue
+			}
+			for _, resultant := range resultants {
+				au := target.AugmentedUnstructured{Object: *resultant, Namespace: &ns}
```

Comment:
Are we giving the generated/not generated metadata to the constraint framework so that the selectors can work there as well?